File: src/main/java/org/dxworks/githubminer/dto/response/repository/pullrequests/PullRequestBranch.java
package org.dxworks.githubminer.dto.response.repository.pullrequests;

import com.google.api.client.json.GenericJson;
import com.google.api.client.util.Key;
import lombok.Data;
import org.dxworks.githubminer.dto.commons.User;
import org.dxworks.githubminer.dto.response.repository.Repository;

@Data
public class PullRequestBranch extends GenericJson {
    @Key
    private String label;
    @Key
    private String ref;
    @Key
    private String sha;
    @Key
    private User user;
    @Key
    private Repository repo;
}

File: github-miner-core/src/main/kotlin/org/dxworks/githubminer/dto/response/repository/pullrequests/PullRequestBranch.kt
package org.dxworks.githubminer.dto.response.repository.pullrequests

import com.google.api.client.json.GenericJson
import com.google.api.client.util.Key
import org.dxworks.githubminer.dto.commons.User
import org.dxworks.githubminer.dto.response.repository.Repository

class PullRequestBranch : GenericJson() {
    @Key
    var label: String? = null

    @Key
    var ref: String? = null

    @Key
    var sha: String? = null

    @Key
    var user: User? = null

    @Key
    var repo: Repository? = null
}
